iiify/resolver.py

```diff
@@ -889,6 +891,29 @@ def create_annotations(version: int, identifier: str, fileName: str, canvas_no:
 
     return json.loads(annotationPage.jsonld())
 
+def create_annotations_from_comments(version, identifier, domain=None):
+    annotation_page = AnnotationPage(
+        id=f"{domain}{version}/annotations/{identifier}/comments.json"
+    )
+    metadata = requests.get('%s/metadata/%s' % (ARCHIVE, identifier)).json()
+    i = 0
+    for comment in metadata.get('reviews', []):
+        anno_body = ResourceItem1(
+            type="TextualBody",
+            language="none",
+            format="text/plain",
+            value=f"<span><p>{comment.get('reviewtitle')}:</p><p>{comment.get('reviewbody')}</p></span>"
+        )
```

Comment:
Is this only version 3?

Reply:
Yes, should be addressed with new changes.